runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -1325,6 +1325,17 @@ def DeleteCommands()
   sign_undefine(BreakpointSigns->map("'debugBreakpoint' .. v:val"))
 enddef
 
+def QuoteArg(x: string): string
+  var x_escaped = x ->substitute('\\', '\\\\', 'g')
+    ->substitute('"', '\\"', 'g')
```

Comment:
Why the quote escape? There can be no quotes in parameter.

Reply:
TBH I sticked to this request https://github.com/vim/vim/pull/15261#issuecomment-2229032515 However, reading at the point made by @LemonBoy above it looks like we need `->substitute('"', '\\"', 'g')` as well.

Reply:
You can condense that into a single `->substitute('[\\"]', '\\\0', 'g')` or something similar.

Reply:
> You can condense that into a single `->substitute('[\\"]', '\\\0', 'g')` or something similar.

Sorry, just read that. I hope we can keep as it is in the last commit. Let me know.